Table.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+export default function Table() {
+
+  return (
+    <table className = "table"> 
+
+        <tr> 
+
+          <th>Name</th> 
+
+          <th>Age</th> 
+
+          <th>Gender</th> 
+
+        </tr> 
+
+        <tr> 
+
+          <td>Anom</td> 
+
+          <td>19</td> 
+
+          <td>Male</td> 
+
+        </tr> 
+
+        <tr> 
+
+          <td>Megha</td> 
+
+          <td>19</td> 
+
+          <td>Female</td> 
+
+        </tr> 
+
+        <tr> 
+
+          <td>Subham</td> 
+
+          <td>25</td> 
+
+          <td>Male</td> 
+
+        </tr> 
```

Comment:
This is a good example, but the component itself has to be general so it will accept any "children". It's actually very simple, just use:

```
{props.children}
```

As for the example data you gave, I think it would be a good example for the story book. You can find an example of how to write an example here:

https://github.com/nenkan/ui/blob/fe23d684977b473f8aedf01b87eaf1c776354904/stories/Header.stories.jsx

You can cut and paste your table content in a new `stories/Table.stories.jsx`

Reply:
Ok thanks.